lib/kitchen/driver/vagrant.rb

```diff
@@ -389,6 +402,12 @@ def update_state(state)
         state[:rdp_port] = hash["RDPPort"] if hash["RDPPort"]
       end
 
+      # @return [String] full absolute path to the kitchen cache directory
+      # @api private
+      def local_kitchen_cache
+        @local_kitchen_cache ||= File.expand_path('~/.kitchen/cache')
```

Comment:
It's actually a bigger problem on Unix systems:

```
$ env HOME=/tmp ruby -e 'puts File.expand_path("~/foo")'
/tmp/foo
```

When running in a jenkins job or similar, it's easy for `$HOME` to not be set correctly since that's usually handled by the login shell. Here be dragons. Might be better to explicitly get the current user's homedir.